cmd/fscrypt/commands.go

```diff
@@ -414,7 +414,7 @@ func unlockAction(c *cli.Context) error {
 	if policy.IsProvisionedByTargetUser() {
 		log.Printf("policy %s is already provisioned by %v",
 			policy.Descriptor(), ctx.TargetUser.Username)
-		return newExitError(c, errors.Wrapf(ErrDirAlreadyUnlocked, path))
+		return newExitError(c, errors.Wrapf(ErrDirAlreadyUnlocked, "path: %s", path))
```

Comment:
Shouldn't these just be replaced with `errors.Wrap()`?

Otherwise the printed message looks weird:

```
[ERROR] fscrypt unlock: path: /test/dir: this file or directory is already unlocked
```

For comparison, currently it is:
```
[ERROR] fscrypt unlock: /test/dir: this file or directory is already unlocked
```

Reply:
https://github.com/google/fscrypt/pull/424 fixes it correctly using errors.Wrap().